src/ontology/uberon-edit.obo

```diff
@@ -42870,8 +42870,8 @@ property_value: taxon_notes "present in all fishes and tetrapods" xsd:string {so
 [Term]
 id: UBERON:0002139
 name: subcommissural organ
-def: "The subcommissural organ is a circumventricular organ consisting of ependymal cells which secrete SCO-spondin[WP,partially vetted]." [PMID:9579598, Wikipedia:Subcommissural_organ]
-comment: Occurs throughout the vertebartes [PMID:9579598, DOI:10.1002/ar.1091260210]. First appears in hagfish(Olsson). In many species, including the human, it reaches its full development during embryonic life[PMID:9579598]. During the course of phyletic evolution, one notes various changes. Indeed, in lower vertebrates, the SCO presents a greater degree of enzyme activity than the ependyma while in birds the ependyma demonstrates a more intense activity than the SCO[PMID:479574] Sometimes preceded by flexural organ[DOI:10.1007/BF00303086]
+def: "The subcommissural organ is a circumventricular organ consisting of ependymal and hypendymal cells which secrete SCO-spondin[WP,partially vetted]." [PMID:9579598, Wikipedia:Subcommissural_organ]
```

Comment:
Consider clarifying that hypendymal cells are rare in non-mammals / add to gloss.
Do axioms need to be updated?

Reply:
We can add 'has part' some 'hypendymal cell' but I'm hesitant due to the rare in non-mammals thing.

Reply:
I share that reservation. This is why I suggest making reference to that in the text def.